pages/tokens/[id].tsx

```diff
@@ -182,5 +183,6 @@ export const getStaticProps: GetStaticProps<TokenDetailProps> = async ({ params
     props: {
       token: token,
     },
+    revalidate: DATA_CACHE_TIME_SECONDS,
```

Comment:
this will ensure the data changes. It will still be performant, users won't experience delays